### PR TITLE
fix #5224: show name of new List in moving caches command

### LINKED ISSUE #5224: Broken string in progress bar for moving caches
cgeo2015-10-05 14 42 05

2015.09.28

### PR DESCRIPTION
**Pull request by rahu**

### CODE REVIEW

File: main/src/cgeo/geocaching/command/AbstractCommand.java [hunk 1]
```diff
@@ -97,6 +97,10 @@ public void execute() {
         task.execute(new Void[] {});
     }
 
+    public final void setProgressMessage(String progressMessage) {
```
Comment (samueltardieu) [inline main/src/cgeo/geocaching/command/AbstractCommand.java]: Still a missing `final` there :-)

File: main/src/cgeo/geocaching/command/MoveToListCommand.java [hunk 2]
```diff
@@ -33,6 +33,8 @@ public void execute() {
             @Override
             public void call(final Integer newListId) {
                 MoveToListCommand.this.newListId = newListId;
+                final String newListName = StoredList.getListById(newListId).getTitle();
+                MoveToListCommand.super.setProgressMessage(getContext().getString(R.string.command_move_caches_progress, newListName));
```
Comment (samueltardieu) [inline main/src/cgeo/geocaching/command/MoveToListCommand.java]: Can't you just call `setProgressMessage`, and omit `MoveToListCommand.super`?

**Comment by rahu** [comment]
and I don't want to merge this into release ...

**Comment by samueltardieu** [comment]
Why not?

**Comment by rahu** [comment]
from https://github.com/cgeo/cgeo/wiki/How-to-get-a-bug-fix-into-the-release : 
"creating their branch off the release branch, and submit their pull request against master"

**Comment by samueltardieu** [comment]
Oh yes :-)

**Pull request closed without merging**